sources/editor/Stride.Assets.Presentation/AssetEditors/EntityHierarchyEditor/ViewModels/EntityHierarchyEditorViewModel.cs

```diff
@@ -496,7 +496,7 @@ protected override async Task Delete()
                     confirmMessage = string.Format(Tr._p("Message", "Are you sure you want to delete these {0} entities?"), entitiesToDelete.Count);
                 var checkedMessage = string.Format(Stride.Core.Assets.Editor.Settings.EditorSettings.AlwaysDeleteWithoutAsking, "entities");
                 var buttons = DialogHelper.CreateButtons(new[] { Tr._p("Button", "Delete"), Tr._p("Button", "Cancel") }, 1, 2);
-                var result = await ServiceProvider.Get<IDialogService>().CheckedMessageBox(confirmMessage, false, checkedMessage, buttons, MessageBoxImage.Question);
+                var result = await ServiceProvider.Get<IDialogService2>().CheckedMessageBox(confirmMessage, false, checkedMessage, buttons, MessageBoxImage.Question);
```

Comment:
I quickly looked into and it looks legit, just don't forget to rename this interface late, because name may be confusing 👀

Reply:
Yeah, not all methods have been ported to the new interface. Some will depend on how they are later implemented with Avalonia to figure out their final signatures.

Eventually, this interface will disappear.